lighthouse-core/audits/metrics.js

```diff
@@ -26,7 +36,7 @@ class Metrics extends Audit {
       scoreDisplayMode: Audit.SCORING_MODES.INFORMATIVE,
       title: 'Metrics',
       description: 'Collects all available metrics.',
-      requiredArtifacts: ['traces', 'devtoolsLogs'],
+      requiredArtifacts: ['traces', 'devtoolsLogs', 'Visibility'],
```

Comment:
this is when the idea of `optionalArtifacts` would be very very handy

Reply:
oh hi, we are in the future now

Reply:
we have a long history and set of other tooling that creates metrics from just a trace and a devtoolslog that has no idea `Visibility` exists and would all need to be updated to create a fake one.

if it's optional then we just progressively enhance with the warning when the data is available